test/bootstrap.php

```diff
@@ -17,3 +17,7 @@
 
 require_once __DIR__ . '/../vendor/autoload.php';
 require_once __DIR__ . '/TestUtils/sleeper.php';
+
+use Google\Cloud\TestUtils\PHPUnit4To6Sim;
```

Comment:
Do you mean `PHPUnit4to6Shim`? Why don't we just call it `PHPUnitShim`? Or have two different ones, `PHPUnit4Shim` and `PHPUnit6Shim`?